local-tests/setup/tinny-environment.ts

```diff
@@ -494,47 +518,53 @@ export class TinnyEnvironment {
       });
     }
 
+    if (!this.contractsClient) {
+      console.log('❗️Contracts client not initialized');
+      process.exit();
+    }
+
     await this.contractsClient.connect();
 
     /**
      * ====================================
      * Mint a Capacity Credits NFT and get a capacity delegation authSig with it
      * ====================================
      */
+    if (CENTRALISATION_BY_NETWORK[this.network] === 'decentralised') {
+      await this.mintSuperCapacityDelegationAuthSig(wallet);
+    }
+  };
 
-    // Disabled for now
-    async function _mintSuperCapacityDelegationAuthSig() {
-      console.log(
-        '[𐬺🧪 Tinny Environment𐬺] Mint a Capacity Credits NFT and get a capacity delegation authSig with it'
-      );
-      try {
-        const capacityTokenId = (
-          await this.contractsClient.mintCapacityCreditsNFT({
-            requestsPerKilosecond: this.processEnvs.REQUEST_PER_KILOSECOND,
-            daysUntilUTCMidnightExpiration: 2,
-          })
-        ).capacityTokenIdStr;
-
-        this.superCapacityDelegationAuthSig = (
-          await this.litNodeClient.createCapacityDelegationAuthSig({
-            dAppOwnerWallet: wallet,
-            capacityTokenId: capacityTokenId,
-            // Sets a maximum limit of 200 times that the delegation can be used and prevents usage beyond it
-            uses: '200',
-          })
-        ).capacityDelegationAuthSig;
-      } catch (e: any) {
-        if (
-          e.message.includes(`Can't allocate capacity beyond the global max`)
-        ) {
-          console.log('❗️Skipping capacity delegation auth sig setup.', e);
-        } else {
-          console.log(
-            '❗️Error while setting up capacity delegation auth sig',
-            e
-          );
-        }
+  async mintSuperCapacityDelegationAuthSig(wallet: Signer) {
+    console.log(
+      '[𐬺🧪 Tinny Environment𐬺] Mint a Capacity Credits NFT and get a capacity delegation authSig with it'
+    );
+
+    const capacityTokenId = (
+      await this.contractsClient.mintCapacityCreditsNFT({
+        requestsPerKilosecond: this.processEnvs.REQUEST_PER_KILOSECOND,
+        daysUntilUTCMidnightExpiration: 2,
+      })
+    ).capacityTokenIdStr;
+
+    try {
+      this.superCapacityDelegationAuthSig = (
+        await this.litNodeClient.createCapacityDelegationAuthSig({
+          dAppOwnerWallet: wallet,
+          capacityTokenId: capacityTokenId,
+          // Sets a maximum limit of 200 times that the delegation can be used and prevents usage beyond it
+          uses: '200',
+        })
+      ).capacityDelegationAuthSig;
+    } catch (e: any) {
+      if (e.message.includes(`Can't allocate capacity beyond the global max`)) {
```

Comment:
:nit: Is it really ok to always ignore all errors from this code? If this code is necessary, and it fails, then why not exit immediately so we know exactly what went wrong? Ideally we wouldn't continue only to have a later test fail.

Reply:
yea - i think it makes sense to just throw the actual error now!